modules/flake-parts/darwinConfigurations.macos-02/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  self,
+  lib,
+  inputs,
+  config,
+  ...
+}: let
+  system = "x86_64-darwin";
+in {
+  flake.darwinConfigurations.macos-02 = inputs.darwin.lib.darwinSystem {
```

Comment:
this currently fails with

```
       Definition values:
       - In `/nix/store/njlbbgnbhkfh6l3cymwr5wv8jcqpxkx9-4pjadv6lxizs52nwvqbhhavq5llsn4hg-source/modules/flake-parts/darwinConfigurations.macos-01'
       - In `/nix/store/njlbbgnbhkfh6l3cymwr5wv8jcqpxkx9-4pjadv6lxizs52nwvqbhhavq5llsn4hg-source/modules/flake-parts/darwinConfigurations.macos-02'
```

cc @DavHau

Reply:
i pushed a fix to this in https://github.com/holochain/holochain-infra/pull/21/commits/6d7ee4ccf8a652fb26408bc586305e6fb17b4164. curious to know whether the solution is idiomatic.

Reply:
This solution is good. This is needed because there is no standard flake output attribute for darwinConfigurations.